src/general/OptionsMenu.tscn

```diff
@@ -248,21 +247,20 @@ size_flags_horizontal = 3
 size_flags_vertical = 3
 theme_override_constants/separation = 10
 
-[node name="Vsync" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Graphics/ScrollContainer/MarginContainer/VBoxContainer" instance=ExtResource("10")]
+[node name="Vsync" type="CheckBox" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Graphics/ScrollContainer/MarginContainer/VBoxContainer"]
 layout_mode = 2
 size_flags_horizontal = 0
 text = "VSYNC"
 
-[node name="Fullscreen" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Graphics/ScrollContainer/MarginContainer/VBoxContainer" instance=ExtResource("10")]
+[node name="Fullscreen" type="CheckBox" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Graphics/ScrollContainer/MarginContainer/VBoxContainer"]
 layout_mode = 2
 size_flags_horizontal = 0
-focus_neighbor_bottom = NodePath("../HBoxContainer/MSAAResolution")
 text = "FULLSCREEN"
 
 [node name="FocusGrabber" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Graphics/ScrollContainer/MarginContainer/VBoxContainer" instance=ExtResource("13")]
 layout_mode = 2
 Priority = 3
-NodeToGiveFocusTo = NodePath("../Vsync")
```

Comment:
This probably broke something, also I see a lot of removed focus lines. It'd probably help to just go through each changed line of this diff and reverting any lines that look even a bit suspiciously like not good changes.

Reply:
Maybe it was this, but I didn't update node type in .cs (I had to turn CustomCheckBox into Checkbox).

Reply:
The type of that variable should be `NodePath` and it should just fetch it as `GetNode<Control>` so the type doesn't matter. But you might be onto something, there is probably some place that specifically wants a custom checkbox type variable and that needs updating.

Reply:
I mean, it works now since I changed from `GetNode<CustomCheckBox>(NodePath)` to `GetNode<CheckBox>(NodePath)`.